tests/templates/sql/nested.html

```diff
@@ -0,0 +1,4 @@
+{% extends "base.html" %}
+{% block content %}
+  {% include "sql/included.html" %}
```

Comment:
How come you went with an `include` here rather than using `{{ users }}`?

Reply:
From the issue #902 

> The SQL query of Question.objects.all() is executed at the line 5 of inner.html, but in the SQL panel, it is incorrectly pointed to the line 5 of its parent template index.html

The problem is that the SQL panel does not show the template context when using included templates. This file to test the correct behavior.

Reply:
I'm sorry, I didn't realize we were in the tests directory.